ee/clickhouse/queries/test/test_person_query.py

```diff
@@ -91,7 +91,7 @@ def test_person_query_with_multiple_cohorts(testdata, team, snapshot):
                         "type": "OR",
                         "values": [
                             {"key": "group", "value": "none", "type": "person"},
-                            {"key": "group", "value": [1, 2, 3], "type": "person"},
+                            {"key": "group", "value": ["1", "2", "3"], "type": "person"},
```

Comment:
Prior to this change, the HogQL query was generating this as a WHERE clause:

```sql
WHERE in(persons.properties___group, tuple(1.0, 2.0, 3.0))
```

Which didn't match anything, causing the test to fail because the Cohort it created was empty. It appears HogQL is a bit more strict around this perhaps and the old code more forgiving.

By changing these to strings, it changes the query to:

```sql
in(persons.properties___group, tuple(%(hogql_val_13)s, %(hogql_val_14)s, %(hogql_val_15)s))
```